src/config/database.js

```diff
@@ -0,0 +1,13 @@
+import Sequelize from "sequelize";
+
+export default new Sequelize('postgres', 'postgres', 'password', {
```

Comment:
Such variables as db url,port, username should never be commited to a repository. This is a secure data, so you should get it from environmental variables. Node allows get env variables in such way: process.env.%YOUR_VARIBALE_NAME%
I recommend to use dotenv lib for convenience: https://www.npmjs.com/package/dotenv